.github/workflows/release-chart.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release prometheus chart
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  #   Sometimes chart-releaser might fetch an outdated index.yaml from gh-pages, causing a WAW hazard on the repo
+  #   This job checks the remote file is up to date with the local one on release
+  validate-gh-pages-index:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+      - name: Download remote index file and check equality
+        run: |
+          curl -vsSL https://newrelic.github.io/nri-prometheus/index.yaml > index.yaml.remote
```

Comment:
Can we use Github magic vars/expressions to get `nri-prometheus` procedurally? I think it would be useful for forks etc.